docs/concepts/storage/persistent-volumes.md

```diff
@@ -171,10 +171,10 @@ parameters:
 allowVolumeExpansion: true
 ```
 
-Once both feature gate and aforementioned admission plug-in are turned on, an user can request larger volume for their `PersistentVolumeClaim`
-by simply editing the claim and requesting bigger size.  This in turn will trigger expansion of volume that is backing underlying `PersistentVolume`.
+Once both feature gate and the aforementioned admission plug-in are turned on, an user can request larger volume for their `PersistentVolumeClaim`
+by simply editing the claim and requesting a larger size.  This in turn will trigger expansion of the volume that is backing the underlying `PersistentVolume`.
 
-Under no circumstances a new `PersistentVolume` gets created to satisfy the claim. Kubernetes will attempt to resize existing volume to satisfy the claim.
+Under no circumstances will a new `PersistentVolume` be created to satisfy the claim—Kubernetes will instead attempt to resize the existing volume.
```

Comment:
nit: leave space before and after the dash? Just a style thing.

Reply:
This is an em dash. Typically it doesn't have spaces around it, unless you're writing for a newspaper/magazine. The repo has 22 instances of no spaces. The 2 times there are spaces are in quotes.

That could honestly be a period too to make it simpler.

Reply:
Alright, I was calling it a nit because ... when the revised text is rendered in my browser, it looks more like a simple minus ('-') rather than a dash. Could be a font setting problem at my side. However, it did ring an alarm to me. I can vaguely recall some suggestions from an experienced doc writer who was against dashes. Not sure if more coffee would be helpful to my RAM.

Reply:
The font GitHub uses in the diff has a very narrow emdash. The rendered markdown is a lot better. A hyphen (-) is definitely wrong, as is an en dash (–). In the interest of simplicity I just opted for a plain old period.